cylc/flow/scripts/lint.py

```diff
@@ -189,15 +189,19 @@ def get_wallclock_directives():
             'TIME_LIMIT_DIRECTIVE',
             None
         )
-        if directive:
-            directives[module.name] = directive
+        if directive and directive == '-W':
+            # LSF directive -W needs to have a particular form to
+            # avoid matching PBS directive -W:
+            directives[module.name] = re.compile(r'^-W\s+(\d+:)?\d+$')
```

Comment:
I think we need to allow for an `=` char here, else we'll get false negatives for LSF directives?